serverless-plugin/index.js

```diff
@@ -145,15 +145,19 @@ class ServerlessPlugin {
 
     assertServerlessPackageVersionsMatch(directory, packageName) {
         // Check that the Haskell package version corresponds to our own
-        const haskellPackageVersion = this.runStackOutput(
+        const stackDependencies = this.runStackOutput(
             directory,
             [
-                'exec', '--',
-                'ghc-pkg',
-                'field', PACKAGE_NAME, 'version',
-                '--simple-output'
+                'ls',
+                'dependencies',
             ]
-        );
+        ).split("\n");
+        const haskellPackageVersions = stackDependencies.filter(dep => dep.startsWith(`${PACKAGE_NAME} `));
+        if (haskellPackageVersions.length === 0) {
+            this.serverless.cli.log(`Could not find ${PACKAGE_NAME} in stack's dependencies.`);
```

Comment:
Can you please add an instruction to fix it? Something like "Please add serverless-haskell to extra-deps in stack.yaml or use LTS 12 or above."

Reply:
Of course! :)

Reply:
I've updated it to be a bit more thorough. Dunno if it's too long now though, but I guess that's better than people not knowing how to resolve it.

Reply:
Sorry, I meant adding something to _this_ line, instructions on what to do if there is no Haskell package installed.

Reply:
Jesus, I don't know how I missed that... I've updated the message now.